pages.es/osx/bless.md

```diff
@@ -7,7 +7,7 @@
 
 `bless --folder {{/Volumes/Mac OS X/System/Library/CoreServices}} --bootinfo --bootefi`
 
-- Configura un volumen que contenga Mac OS 9 y Mac OS X para que sea el volumen activo:
+- Configura un volumen que contenga macOS 9 y Mac OS X para que sea el volumen activo:
```

Comment:
```suggestion
- Configura un volumen que contenga Mac OS 9 y Mac OS X para que sea el volumen activo:
```

Apparently, [names with the string `macOS` came afterwards](https://en.m.wikipedia.org/wiki/MacOS_version_history) so I am guessing the correct name is `Mac OS 9`

Reply:
I wonder if this command and the others don't work for newer versions of macOS